.github/ISSUE_TEMPLATE/new-article.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New Article
+about: Pitch an idea for an article or claim one to write yourself.
+title: "Article: [Your title]"
+labels: "[Content] Article"
+
+---
+
+Delete this boilerplate or any other information that doesn't apply to your article.
+Please review our [contributing guidelines](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#submitting-content).
+
+## Description
+>A brief description (2-3 sentences). Why should this article be written and how does it fit on a11yproject.com? Please also include problems you (the person filing the issue) have faced concerning this topic.
```

Comment:
>Why should this article be written, and how does it fit on The A11Y Project?

Added a comma and used the formal treatment for our site name.

Reply:
>Please also include problems you (the person filing the issue) have faced concerning this topic.

Could we rephrase this? I don't want to make it seem like you have to have faced accessibility-related issues to be able to write an article (this might exclude a dev writing about their experiences learning about accessibility, for example). 

Maybe suggest something along the lines of how personal experiences can make for a great starting point for an article?

Reply:
I agree about the last description sentence - I think this is ultimately unnecessary so I removed the sentence completely. I think the links to the contrib guidelines and the "why should it be written" do the same thing.